src/client/components/informationPanel/InformationSocials.js

```diff
@@ -0,0 +1,151 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Text, Card, theme, Button, IconClose } from '@aragon/ui'
+import {
+  Social,
+  Link,
+  IconGitHub,
+  IconTwitter,
+  IconVerified,
+} from '../styled-components'
+
+const PopoverCard = ({ social, setPopover }) => {
+  return (
+    <VerifyCard>
+      <Text.Block size="xlarge">Verify my {social}</Text.Block>
+      <CardCloseButton
+        type="button"
+        onClick={() => {
+          setPopover('')
+        }}
+      >
+        <IconClose />￼
+      </CardCloseButton>
+
+      <Text.Block size="small">
+        Aragon manages profiles using 3box.io. To verify your {social}, you must
+        visit your 3box profile.
+      </Text.Block>
+      <Link size="small" href="https://3box.io">
+        Take me to my 3box
+      </Link>
+    </VerifyCard>
+  )
+}
+
+PopoverCard.propTypes = {
+  social: PropTypes.string,
+  setPopover: PropTypes.func.isRequired,
+}
+
+export const RenderGitHub = ({
+  github: { username },
+  activePopover,
+  setPopover,
+}) => (
+  <Social>
+    <IconGitHub width="13px" height="13px" color={theme.textTertiary} />
+    {username ? (
+      <div>
+        <Link href={`https://github.com/${username}`}>{username}</Link>
+        <Verified />
+      </div>
+    ) : (
+      <div>
+        <Button
+          compact
+          mode="outline"
+          css={`
+            position: relative;
+          `}
+          onClick={() => setPopover('github')}
+        >
+          Verify my GitHub account
+        </Button>
+        {activePopover === 'github' && (
+          <PopoverCard social="GitHub" setPopover={setPopover} />
```

Comment:
Instead of passing `activePopover` and `setPopover`, I _think_ we could 

1. `const [open, setOpen] = useState(false)` in each of these components
2. make sure the popover closes when the user clicks outside of it

Can you think of a problem with that approach?

Reply:
I tried that (seemed much more logical), unfortunately state needs to be passed down from a higher level component.

Reply:
hmm... do you know why state had to be passed down? What problem did you run into, using `useState` in this component?